### PR TITLE
Change fork to newly structured repo for our needs

### PR DESCRIPTION
**Pull request by jebeck**
What it says on the tin.

### CODE REVIEW

**Comment by jebeck** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;8F7bDivf&#x2F;17-redux-blip-uploader-publish-our-fork-of-redux-immutable-state-invariant-to-npm-as-a-object-invariant-test-helper"> [redux] [blip] [uploader] publish our fork of `redux-immutable-state-invariant` to npm as a `object-invariant-test-helper`</a>

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+language: node_js
```
Comment (jebeck) [inline .travis.yml:2]: Oh want to add `sudo: false` at the top here? That allows faster builds on Travis I recently learned.

File: README.md [hunk 2]
```diff
@@ -1,42 +1,20 @@
-# redux-immutable-state-invariant
+# object-invariant-test-helper
 
-Redux middleware that spits an error on you when you try to mutate your state either inside a dispatch or between dispatches. **For development use only!**
+A module that is useful for when needing to test for immutability.
 
-## Why?
+This module began life as a fork of redux-immutable-state-invariant by Leonardo Andres Garcia Crespo.
```
Comment (jebeck) [inline README.md]: Put a link behind this credit to his GitHub profile? And a link behind the `redux-imm...` to the original project?

File: README.md [hunk 3]
```diff
@@ -1,42 +1,20 @@
...
-npm install --save-dev redux-immutable-state-invariant
+npm install --save-dev object-invariant-test-helper
 ```
 
-## How to use
-
-As said above, **don't use this in production!** It involves a lot of object copying and will degrade your app's performance. This is intended to be a tool to aid you in development and help you catch bugs.
+## Usage
 
-To use it, just add it as a middleware in your redux store:
```
Comment (jebeck) [inline README.md:23]: Let's throw an example test in here.

File: package.json [hunk 4]
```diff
@@ -1,24 +1,18 @@
...
   "scripts": {
     "clean": "rm -rf dist",
     "build": "babel --out-dir dist src",
     "test": "mocha --compilers js:babel/register",
-    "prepublish": "npm run clean && npm run build",
-    "example": "cd example && rm -rf node_modules/redux-immutable-state-invariant && npm install && npm start"
+    "prepublish": "npm run clean && npm run build"
   },
-  "author": "Leonardo Andres Garcia Crespo <leoasis@gmail.com>",
+  "author": "Jana Beck <jana@tidepool.org>",
```
Comment (jebeck) [inline package.json]: Hmm, I think npm has a ["contributors" alternative](https://docs.npmjs.com/files/package.json#people-fields-author-contributors), maybe we should use that to include original author? And add you if you like ;)

File: README.md (hunk 3 above)
Comment (GordyD) [inline README.md:23]: I'm not sure what this looks like yet, haven't had a chance to test. Do you have a snippet?

File: README.md (hunk 2 above)
Comment (GordyD) [inline README.md]: Ok!

File: .travis.yml (hunk 1 above)
Comment (GordyD) [inline .travis.yml:2]: Sure

File: package.json (hunk 4 above)
Comment (GordyD) [inline package.json]: Yea, good idea, on it!

File: README.md [hunk 5]
```diff
@@ -1,42 +1,41 @@
...
+    let initialState = {num: 1};
+    const tracked = stateInvariantTestHelper.trackObj(initialState);
+
+    let finalState = goodReducer(initialState, {
+      type: 'ADD',
+      payload: { toAdd }
+    });
+
+    expect(finalState).toEqual(expectedState);
+    expect(stateInvariantTestHelper.hasMutated(tracked)).toEqual(false);
```
Comment (GordyD) [inline README.md]: Looks good. This API feels a lot cleaner to me.

**Pull request merged**